### ISSUE TITLE
Question: Is MathCAT meant to be adopted by AT tools (like NVDA, JAWS, etc.)

### ISSUE THREAD

**Issue by brichwin** [NSoiffer/MathCAT#5]
Hi,

I'm curious if MathCAT could be directly called by AT like the JAWS screen-reading software. JAWS is not reading MathML in a clear, unambiguous manner (I filed an issue on this here: https://github.com/FreedomScientific/VFO-standards-support/issues/539 ). 

Rather than AT providers trying (and failing) to do this work on their own, will MathCAT have an API so AT providers can call it instead of their being a universe full of varied mathematics to speech implementations? I would think that would foster consistency which would greatly improve the easy of comprehension. 

Are their own going efforts to have MathCAT adopted by JAWS in particular? I'd like to supplement my bug report to Freedom Scientific with a suggestion to adopt MathCAT if that is appropriate.

Thanks for all the great work!

**Comment by NSoiffer** [comment]
I have had some conversations with someone connected to JAWS and they said they were interested in *maybe* using it. It has interfaces for C and Python, and so should be able to work with JAWS, NVDA, and Orca (Linux). I'm wrapping up work (hopefully this week) on a Nemeth implementation. It is (already) much less buggy than the liblouis implementation which I believe JAWS uses for Nemeth. I hope that is an extra incentive for them to take up. I plan to submit the tests I've developed for MathML->Nemeth to liblouis and get an actual number on the number of bugs they have on that test. Once I have that, I'll contact them again and point out the improvement they can get from the switch.

It took years of cajoling to get them to implement math speech, so don't expect rapid action for them to switch.

**Comment by NSoiffer** [comment]
[just saw this in my drafts folder -- maybe still relevant]

Brian,

I have had some conversations with someone connected to JAWS and they said
they were interested in maybe using it. It has interfaces for C and Python,
and so should be able to work with JAWS, NVDA, and Orca (Linux). I'm
wrapping up work (hopefully this week) on a Nemeth implementation. It is
much less buggy than the liblouis implementation which I believe JAWS uses
for Nemeth. I hope that is an extra incentive for them to take up.

Apple will almost certainly ignore MathCAT.

    Neil
